src/pages/CraftPageAdminPage/CraftToolBox.tsx

```diff
@@ -104,6 +103,7 @@ const BasicToolbox: React.FC = () => {
       <CraftTool
         as={CraftElement.CraftImage}
         width={400}
+        height={149}
         customStyle={{ backgroundImage: `url("https://static.kolable.com/images/default/craft/image.png")` }}
       />
```

Comment:
maybe img will be changed, try this to get height and width

```js
  const img = new Image();
  img.onload = function() {
    alert(this.width + 'x' + this.height);
  }
  img.src = 'https://static.kolable.com/images/default/craft/image.png';
```

Reply:
Adjusted.